src/formio/components/currency.ts

```diff
@@ -0,0 +1,29 @@
+import {InputComponentSchema, MultipleCapable, PrefillConfig} from '..';
+
+type Validator = 'required' | 'min' | 'max';
+type TranslatableKeys = 'label' | 'description' | 'tooltip' | 'suffix';
+
+export type CurrencyInputSchema = InputComponentSchema<number, Validator, TranslatableKeys>;
+
+/**
+ * @group Form.io components
+ * @category Base types
+ */
+export interface BaseCurrencyComponentSchema
+  extends Omit<CurrencyInputSchema, 'hideLabel'>,
+    PrefillConfig {
+  type: 'currency';
+  // additional properties
+  currency?: 'EUR';
```

Comment:
```suggestion
  currency: 'EUR';
```
more strict more better :)